### PR TITLE
PS-2115 fix empty table_files bug and update IM for changed_since update

### PR DESCRIPTION
**Pull request by pivnicek**
- fixes the bug where if `use_file_storage_only` was set and `table_files` was empty, it would throw an error.

- also updates IM to support `changed_since: some time` function for files

### CODE REVIEW

File: src/Docker/Configuration/Container.php [hunk 1]
```diff
@@ -79,7 +79,17 @@ public function getConfigTreeBuilder()
...
-
+        // validation of use_only_file_storage with table_files
+        $root->validate()
+            ->ifTrue(function ($v) {
+                if (!empty($v['runtime']['use_file_storage_only']) && empty($v['storage']['output']['table_files'])) {
+                    return true;
+                }
+                return false;
+            })
+            ->thenInvalid('table_files must be specified for use_file_storage_only_mode')
```
Comment (odinuv) [inline src/Docker/Configuration/Container.php:90]: and must it? like why? what if there is no output?

Comment (pivnicek) [reply src/Docker/Configuration/Container.php:90]: yeah, it must be set, but it can be empty,  let me just double-check that that passes

Comment (pivnicek) [reply src/Docker/Configuration/Container.php:90]: or maybe there shouldn't be any validation required, and the check can go in the implementation.  I'll take another quick look at it

Comment (odinuv) [reply src/Docker/Configuration/Container.php:90]: > yeah, it must be set

in that case it doesn't have to be set, the entire `storage` node is optional

Comment (pivnicek) [reply src/Docker/Configuration/Container.php:90]: ok, I'll take it out, thanks

**Comment by pivnicek** [comment]
yes, should I split it up?

**Pull request merged**